### ISSUE TITLE
Pinokio broken after installing comfyui ai and stable video diffusion

### ISSUE THREAD

**Issue by mertakbal** [pinokiocomputer/pinokio#63]
I tried to install the new comfyui ai with stable video diffusion. Pinokio kept me telling that the installion is complete after downloading the models for SDVideo. But the "install" button kept appearing instead of "start" button at the starting page.
Afterwards I tried to install the "stable video diffusion for NVIDIA" app on Pinokio. Now everything is broken and I just get a blank white page when I open Pinokio.
How can I uninstall and or reset Pinokio, without losing my previous SD webui installation, which was working perfectly? 

OS Ubuntu 22.04

**Comment by lustfeind** [comment]
Same problem here on Windows 10.

**Comment by mertakbal** [comment]
yes. I tried also on windows 10. Pinokio is completely broken and cant start even after rebooting the computer.
